### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by dee115** [cedricp/ddt4all#1203] [Q&A]
Hello, I went to this window and clicked on the command key off / key on restart and the screen became completely black. Can anyone help me please?

![photo_5924554424875864540_y](https://github.com/user-attachments/assets/7114b773-e465-46fd-9492-e121baa88d2b)
![photo_5924554424875864555_y](https://github.com/user-attachments/assets/8eaba7b9-e02c-4d4a-9c8e-5a3cb2cf1942)

**Comment by AndyIreland** [comment]
are you sure it was not the 'soft reset' button you pressed instead? 

try press the 'Reset Hard' button there and see if rlink2 re-boots OK and screen comes back - at this stage it shouldnt hurt.

**Comment by dee115** [comment]
Excuse me, where is this button you mean?

**Comment by dee115** [comment]
> are you sure it was not the 'soft reset' button you pressed instead?
> 
> try press the 'Reset Hard' button there and see if rlink2 re-boots OK and screen comes back - at this stage it shouldnt hurt.

Excuse me, where is this button you mean?

**Comment by AndyIreland** [comment]
> > are you sure it was not the 'soft reset' button you pressed instead?
> > try press the 'Reset Hard' button there and see if rlink2 re-boots OK and screen comes back - at this stage it shouldnt hurt.
> 
> Excuse me, where is this button you mean?

![image](https://github.com/user-attachments/assets/b9862ae0-ecc0-411e-9f18-2e1009d3a456)
